app/views/questionnaire/components/universityDegreeCategory.scala.html

```diff
@@ -6,9 +6,9 @@
 <section>
     @selectValueDifferentFromDisplayedText(questionnaireForm("universityDegreeCategory"),
         models.view.questionnaire.UniversityDegreeCategory.seq.toList)(
-        "-- Select one --") {
+        "Select one") {
         @Messages("universityDegreeCategory.question")
-    }
+    } { }
```

Comment:
do we need this extra paranthesis { }, why not make the hint param default to {} in selectValueDifferentFromDisplayedText